packages/varlet-ui/src/avatar-group/AvatarGroup.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <div :class="classes(n(), [vertical, n('--column'), n('--row')])" :style="rootStyles">
+  <div
+    :class="classes(n(), [hoverable, n('--hover-able'), null], [vertical, n('--column'), n('--row')])"
```

Comment:
I think null is not necessary.
![image](https://github.com/varletjs/varlet/assets/70570907/873882a8-6acd-40ef-ac3b-91f7317a1bb8)